src/components/Item/ItemHeader.js

```diff
@@ -1,28 +1,59 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 
-export const HEADER_HEIGHT = 45;
+import { sGetIsEditing } from '../../reducers/editDashboard';
+import { acRemoveDashboardItem } from '../../actions/editDashboard';
+import DeleteItemButton from './DeleteItemButton';
+
+import classes from './styles/ItemHeader.module.css';
 
 const ItemHeader = props => {
-    const { title, actionButtons, editMode } = props;
+    const {
+        title,
+        editMode,
+        actionButtons,
+        itemId,
+        acRemoveDashboardItem,
+    } = props;
+
+    const handleDeleteItem = () => acRemoveDashboardItem(itemId);
 
     return (
-        <div className="dashboard-item-header">
-            <div
-                className="dashboard-item-header-title"
-                style={{ userSelect: editMode ? 'none' : 'text' }}
-            >
-                {title}
-            </div>
-            {actionButtons}
+        <div className={classes.itemHeaderWrap}>
+            <p className={classes.itemTitle}>{title}</p>
+            {editMode ? (
+                <div className={classes.itemActionsWrap}>
+                    <DeleteItemButton onClick={handleDeleteItem} />
```

Comment:
Is it possible to add a `title` attribute to this `button`, just until our Tooltip components are ready? `title="Delete item"` would help an unsure user understand this icon on hover.

Reply:
`title="Delete item" ` + i18n 🙂🚀